src/core/RecyclerListView.tsx

```diff
@@ -242,6 +242,28 @@ export default class RecyclerListView<P extends RecyclerListViewProps, S extends
         }
     }
 
+    public bringToFocus(index: number, animate?: boolean): void {
```

Comment:
@anirudhagarwal365 can you also add documentation around this in readme and as a comment?

Reply:
Sure @naqvitalha